documentation/website/docs/installation.md

```diff
@@ -86,6 +86,40 @@ $ echo 'PATH="/path/to/pyre-check/scripts:$PATH"' >> ~/.bashrc
 $ source ~/.bashrc
 ```
 
+## Building from Docker
+
+If you're having issues setting up or your OS is not yet supported, you can also use a Docker image. It runs [Debian GNU/Linux 10 (buster)](https://www.debian.org/) and builds pyre-check from source. 
+
+Before starting, ensure that [Docker](https://docs.docker.com/get-docker/) is installed on your computer.
+
+1. Clone the pyre-check repository and navigate to the root directory.
+   ```bash
+   git clone https://github.com/facebook/pyre-check.git
+   cd pyre-check
+   ```  
+
+2. Build the Docker image with the tag `pyre-check` (or another tag if you wish)
+   ```bash
+   docker build -t pyre-docker .
+   ```   
+
+3. Run the new image in a new container `pyre-container` (or another name if you wish)
+   ```bash
+   docker run \                           
+   --name pyre-container \
+   -v /path/to/your/directory:/src \
+   -t -i \
+   pyre-check /bin/bash
+   ```
+
+4. Inside the container, run any pyre-check command now with `pyre`!
+
+   *Note: When initializing pyre-check with `pyre init`, you may have to enter the following paths for the binary and typeshed:*
```

Comment:
Shouldn't these have been set in the dockerfile? Are these still required here?

Reply:
I think you have to set it up in the Dockerfile but then it asks for it again when initializing pyre?

Reply:
Ugh, we should fix that on the Pyre side, but that's out of scope here